src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -1017,13 +1017,15 @@ public void setRootDv(boolean rootDv) {
         this.rootDv = rootDv;
     }
 
-    public boolean isDebug() {
-        return (debug && session.getUser().isSuperuser())
+    public Boolean getDebug() {
```

Comment:
why not still isDebug?

Reply:
Because with isDebug I get this error:

`/dataverse.xhtml @47,87 value="#{SearchIncludeFragment.debug}": The class 'edu.harvard.iq.dataverse.search.SearchIncludeFragment' does not have a readable property 'debug'.`
`javax.el.PropertyNotFoundException: /dataverse.xhtml @47,87 value="#{SearchIncludeFragment.debug}": The class 'edu.harvard.iq.dataverse.search.SearchIncludeFragment' does not have a readable property 'debug'.`